hw/ip/csrng/data/csrng_testplan.hjson

```diff
@@ -7,13 +7,14 @@
                      "hw/dv/tools/dvsim/testplans/intr_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/alert_test_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
+                     "csrng_sec_cm_testplan.hjson"]
   testpoints: [
     {
       name: smoke
       desc: '''
-	    Verify sending instantiate/generate cmds via SW path.
-	    Verify reading genbits via SW path.
+      Verify sending instantiate/generate cmds via SW path.
+      Verify reading genbits via SW path.
```

Comment:
Hmm, these lines got inadvertently modified because I have a "replace preceding tabs with spaces on save" macro enabled on my vim. Perhaps ok to let this slide?

Reply:
If you have to...